src/argilla/server/models/questions.py

```diff
@@ -29,15 +29,15 @@ class ResponseValue(Protocol):
 
 
 class BaseQuestionSettings(BaseModel):
-    def check_response(self, response: ResponseValue):
+    def check_response(self, response: ResponseValue, status: Optional[ResponseStatus] = None):
```

Comment:
@frascuchon I think adding the `status` argument so partial validations depending on the response status can be done. What do you think?